tools/cuds-prioritized-attribution/test/test_file_based_comparision.py

```diff
@@ -12,33 +12,48 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from helper import Helper
 import filecmp
-import pytest
 import os
 import logging
+import pytest
+from helper import project_label_credit, execute_query, create_dataset, prepare_consolidated_billing, clean, dump_result
+import configparser
+
+config = configparser.ConfigParser(allow_no_value=True)
+config.read("pytest.properties")
 
 data = {
-    'project_id': os.environ['project_id'],
-    'corrected_dataset_id': os.environ['corrected_dataset_id'],
-    'corrected_table_name': '',
-    'project_label_credit_breakout_table': '',
-    'distribute_commitments_table': '',
-    'billing_export_dataset_id': os.environ['billing_export_dataset_id'],
-    'billing_export_table_name': '',
-    'load_billing_export_table_name': '',
-    'commitment_table_name': '',
-    'enable_cud_cost_attribution': os.environ['enable_cud_cost_attribution']
+    'project_id':
+        config.get("FILE_BASED_TEST", "project_id"),
+    'corrected_dataset_id':
+        config['FILE_BASED_TEST']['corrected_dataset_id'],
+    'corrected_table_name':
+        '',
+    'project_label_credit_breakout_table':
+        '',
+    'distribute_commitments_table':
+        '',
+    'billing_export_dataset_id':
+        config['FILE_BASED_TEST']['billing_export_dataset_id'],
+    'billing_export_table_name':
+        '',
+    'load_billing_export_table_name':
+        '',
+    'commitment_table_name':
+        '',
+    'enable_cud_cost_attribution':
+        (config['FILE_BASED_TEST']['enable_cud_cost_attribution'] == 'true'),
+    'cud_cost_attribution_option':
+        'a'
 }
-
 test_directory = next(os.walk("tests/"))[1]
 test_directory.sort()
 
 print("\n" + 'Preparing test environment ... ' + "\n")
-Helper.create_dataset(data['corrected_dataset_id'])
+create_dataset(data['corrected_dataset_id'])
 
 
-@pytest.mark.parametrize("dir", test_directory)
+@pytest.mark.parametrize("dir", ["test_7_3", "test_7_4", "test_7_5", "test_7_6"])
```

Comment:
typo?

Reply:
yes ... sorry I was selectively fixing the data. I have pushed the change to make it test_directory again.

Reply:
Thanks. Merged!